Docs cleanup

EduKav1813 opened the pull request.

macpijan commented on docs/transparent-validation/rte/v1.1.0/flashing-guide.md:
```diff
@@ -1,88 +0,0 @@
-# Flashing guide
```
@EduKav1813 You are simply removing this, have you confirmed this is not needed?

@mkopec do we need this doc, or are we fine with what we have under NovaCustom already?

EduKav1813 replied:
This may be already covered in https://docs.dasharo.com/unified/novacustom/recovery/, hence I removed it. But it is a good idea to cross-check.

@dawidlesiak Since you use the docs actively, we would value your input on this issue.

macpijan replied:
I do  not think Dawid uses this doc.
I can ask Wessel from NovaCustom.

pietrushnic replied:
@macpijan @EduKav1813 I asked Wessel on #novacustom-exernal channel. I will let you know what is the answer.
Since I'm an original poster of the issue, I could also do a review. I was not pinged enough to jump into this.

pietrushnic replied:
@EduKav1813 this documentation is still used. Please do not remove it.

pietrushnic commented on docs/transparent-validation/rte/revision-history.md:
```diff
@@ -1,101 +1,178 @@
+[Repository](https://github.com/3mdeb/rte-schematics)
 
-## v1.0.0
+## v. 1.1.0
```
@EduKav1813 why you change this?

pietrushnic replied:
> @pietrushnic To be honest I don't remember at this point. Now I would change it to be close to the source as possible.

Please revert it. There should be no change in headers.

pietrushnic replied:
Pleas keep formatting as close to previous version as possible.

EduKav1813 replied:
Outdated change - please see latest commits

pietrushnic commented on mkdocs.yml:
```diff
@@ -562,6 +561,7 @@ nav:
               - '[ERR] SATA LED and PC speaker error indication': unified-test-documentation/dasharo-compatibility/31S-sata-led-and-pc-speaker-error-indication.md
               - '[ESP] ESP scanning': unified-test-documentation/dasharo-compatibility/361-esp-scanning.md
               - '[FAN] Fan speed': unified-test-documentation/dasharo-compatibility/S30-fan-speed.md
+              - '[FAN] coreboot Fan control': unified-test-documentation/dasharo-compatibility/S31-coreboot-fan-control.md
```
@EduKav1813 I have idea why we change this, how is this related?

EduKav1813 replied:
Outdated change - please see latest commits

EduKav1813 commented:
@pietrushnic This PR was severly outdated, so I remade it on a newest version. What it has now:

- Added changelog from `v1.1.1` of RTE schematic: https://github.com/3mdeb/rte-schematics/pull/8
- Rewrote the `docs/transparent-validation/rte/v1.1.0/flashing-guide.md` to describe general process of platform recovery.

What is potentially left to do:
- Change the `v1.1.0` -> `v1.1.1` across the entire documentation. Although I am not sure that it is within the scope of this PR. Please let me know. Also please note that the PR https://github.com/3mdeb/rte-schematics/pull/8 is still not merged.
